projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts

```diff
@@ -84,6 +84,18 @@ export function checkElement(tabElement: TabElement) {
         .should('have.attr', 'formcontrolname', tabElement.value);
       break;
     }
+    case TabbingOrderTypes.H3: {
+      cy.focused()
+        .get('h3')
+        .should('contain', tabElement.value);
+      return;
+    }
+    case TabbingOrderTypes.CX_MEDIA: {
```

Comment:
can we check something different than images' src? every change to the backend will brake it. too fragile imho

Reply:
Like what?

Reply:
I just noticed that the `<img />` elements have `alt` attributes, which contain name of the product, so we can go this way. much safer tbh